Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs

```diff
@@ -107,9 +107,18 @@ private Vector2 GetScreenResoultion()
         private Vector2 _dragWindowStartPosition;
 #endif
 
+        private void EnsureCorrectScreenSizing()
+        {
+#if !UNITY_2017_4_OR_NEWER
```

Comment:
What about 2017.4 and later?

Reply:
Well, according to our information the issue doesn't appear in versions later or equal to 2017.4.

But then again, maybe we should correct for it anyway.

Reply:
I think we should.